client/canonical_eventlog.go

```diff
@@ -0,0 +1,266 @@
+package client
+
+import (
+	"bytes"
+	"crypto"
+	"encoding/binary"
+	"fmt"
+	"io"
+
+	"github.com/google/go-tpm/tpm2"
+)
+
+var hashAlgoMapping = map[crypto.Hash]tpm2.Algorithm{
```

Comment:
optional: I prefer map names that describe the key and value: something like `cryptoHashToTpmAlg`

Reply:
Doen